crates/router/src/services.rs

```diff
@@ -1,6 +1,6 @@
 pub mod api;
 pub mod authentication;
-#[cfg(feature = "basilisk")]
+// #[cfg(feature = "basilisk")]
```

Comment:
remove commented code, why we are moving out from basilisk feature?

Reply:
this makes the entire encryption module non usable in local, we have fn like jwe, jws which works the same in every env